src/clang-format-all.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Given that this is primarily a Windows product, we need to have solutions that work for Windows users.  For scripting, this would typically mean PowerShell.  We're asking that you move this script to the following location:
```
\Tools\Scripts\clang-format\clang-format-all.sh
```

After we merge your change, we will add a `clang-format-all.ps1` to the same location.  Then there should be a consistent location for any user that needs this functionality.    You'll have to update your script to correctly format files under the `/src/` directory.  It would also then make sense for the `.clang-format` file to move to the repo root.

Reply:
@danbelcher-MSFT all was done. :)